### PR TITLE
bumped version of hive-jdbc in pom

### PR DESCRIPTION
**Pull request by sshetty0705**
Please review.
Thanks.
SS

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Changed
 - Upgraded `jackson-databind` to 2.9.7 (was 2.9.0)
+- Upgraded `hive-jdbc` to 3.1.1 (was 2.0.0)
 
-## [1.0.0] - 2018-10-30
+## [1.0.0] - 2018-11-22
```
Comment (massdosage) [inline CHANGELOG.md]: We've already released 1.0.0 so you can't change its release date.

File: pom.xml [hunk 2]
```diff
@@ -163,7 +163,7 @@
     <dependency>
       <groupId>org.apache.hive</groupId>
       <artifactId>hive-jdbc</artifactId>
-      <version>2.0.0</version>
+      <version>3.1.1</version>
```
Comment (massdosage) [inline pom.xml]: I wouldn't upgrade to this verison of Hive as it's potentially backwards incompatible with previous versions of Hive and we're not using it in production anywhere.

**Review by massdosage — CHANGES_REQUESTED** [review]
I'm assuming you're doing this due to the Hive vulnerabilities reported on GitHub. Do any of them actually apply to how Quibble uses these libraries? We dismissed them on Circus Train and other projects since we don't use Hive in the way the vulnerability is manifested.

**Comment by sshetty0705** [comment]
Will reopen the pull request, closing this one.

**Pull request closed without merging**